### PR TITLE
Add VAT validation rule #5 [VS-16]

### PR DESCRIPTION
**Pull request by rsasch**
Closes https://broadworkbench.atlassian.net/browse/VS-16 by adding SQL for
> There is a non-zero number of transcript fields with null values in the VAT.

### CODE REVIEW

File: scripts/variantstore/wdl/GvsValidateVAT.wdl [hunk 1]
```diff
@@ -245,7 +258,7 @@ task SchemaOnlyOneRowPerNullTranscript {
             transcript_source is NULL AND
             transcript is NULL
         GROUP BY vid
-        HAVING num_rows = 1' > bq_variant_count.csv
+        HAVING num_rows > 1' > bq_variant_count.csv
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsValidateVAT.wdl:261]: nice catch!

Comment (RoriCremer) [reply scripts/variantstore/wdl/GvsValidateVAT.wdl:261]: nit: while you are here, there's a tiny typo below:
"# if the result of the query has any rows, that means there were vids will null transcripts and multiple"
 I think will should be with?

**Review by RoriCremer — APPROVED** [review]
What does the 'import "GvsWarpTasks.wdl" as Tasks' do?

**Comment by rsasch** [comment]
converting to draft because `SchemaNonzeroAcAn` is failing

**Comment by RoriCremer** [comment]
ah---this is what I get for making an example inputs file when we really didn't need one and for choosing the full AoU 1k release (vat_kc_vat_1) as the default. I wanted to run the validations here because it is the largest dataset and it is the AoU data (not just Anvil data) BUT it has no values for gvs_all_ac or gvs_all_an yet because that step wasn't implemented by the time of creation.  (Validation #9 was added by Lee recently) The table vat_jul18 does have those values as it was created just last week, but may get cleaned up...so this might be a good best practices question for what we run this on in the future if there is ever an automated version?

**Pull request merged**